integration-tests/tests/e2e.rs

```diff
@@ -345,7 +346,7 @@ where
 
     // Execute TO1/TO2 protocols
     let ssh_authorized_keys_path = ctx.testpath().join("authorized_keys");
-    let marker_file_path = ctx.testpath().join("marker");
+    let marker_file_path = PathBuf::from("/etc/device_onboarding_performed"); //setting the marker file default value from clientlinuxapp
```

Comment:
this is gonna use the system, why changing this?

Reply:
So if I run the test on my system that file is going to show up?

Reply:
yes, it will, and you have to `rm' the file before running another test for the onboarding process to run.

Reply:
I don't understand the difference between the mocked marker file and the default path - they do the very same, what is this change actually doing?

Reply:
the problem here is mainly around running privileged tests in environments like local dev - maybe we just need to skip privileged tests @puiterwijk ? 👼

Reply:
@runcom More I think there can be a way move into the the file location to any custom path, but creating user itself require privileges.

Reply:
if this test requires privileges we need to wrap it under some condition so it's not run on my local dev environment cause I don't want an user to be run in my local laptop - CI can expose an env variable to say "I'm in CI" and then this test is run only under that condition

Reply:
@runcom  @nullr0ute updated the test accordingly.